components/Shell.tsx

```diff
@@ -192,6 +193,16 @@ export default function Shell(props) {
 }
 
 function UserDropdown({ session, small, bottom }: { session: any; small?: boolean; bottom?: boolean }) {
+  const [avatarSrc, setAvatarSrc] = useState<string>("");
+
+  useEffect(() => {
+    fetch("/api/me")
+      .then((res) => res.json())
+      .then((user) => {
+        setAvatarSrc(user.avatar);
+      });
+  }, []);
```

Comment:
This isn't ideal but since we aren't using `react-query` or `swr` it's really tricky to write those nicely, so it's OK as-is

Reply:
thoughts on using react-query or swr? I like swr a lot